toolkit/tools/imager/imager.go

```diff
@@ -474,11 +474,19 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap m
 
 func configureDiskBootloader(systemConfig configuration.SystemConfig, installChroot *safechroot.Chroot, diskDevPath string, installMap map[string]string, encryptedRoot diskutils.EncryptedRootDevice) (err error) {
 	const rootMountPoint = "/"
+	const bootMountPoint = "/boot"
 
 	var rootDevice string
 
-	// Add bootloader
-	bootUUID, err := installutils.GetUUID(installMap[rootMountPoint])
+	// Add bootloader. Prefer a seperate boot partition if one exists.
+	bootDevice, ok := installMap[bootMountPoint]
```

Comment:
>ok [](start = 13, length = 2)

Nit: the name "ok" sends a misleading message that there' something wrong, if this value is not present inside `bootMountPoint`. What do you think about "bootMountPointPresent"?

Reply:
The 'ok' is VERY golang (agreed its a bit silly though).

Reply:
Its the standard way of checking for the presence of a value in a map